connaisseur/tests/test_image.py

```diff
@@ -57,6 +57,7 @@ def im():
             "path",
             "sub.registry.io",
         ),
+        ("registry:3000/image:tag", "image", "tag", None, "", "registry:3000"),
```

Comment:
maybe a test case for the error case (`registry/image:tag`)?

Reply:
should be included here:
https://github.com/sse-secure-systems/connaisseur/blob/aee751bcd87232c2f92f4b6a1178799ccee357c9/connaisseur/tests/test_image.py#L42